dMasterServer/MasterServer.cpp

```diff
@@ -154,15 +155,19 @@ int main(int argc, char** argv) {
 
 		Game::logger->Log("WorldServer", "Found cdclient.fdb.  Converting to SQLite");
 
-		if (FdbToSqlite::Convert(Game::assetManager->GetResPath().string()).ConvertDatabase() == false) {
+		if (FdbToSqlite::Convert(Game::assetManager->GetResPath().string(), (BinaryPathFinder::GetBinaryDir() / "resServer").string()).ConvertDatabase() == false) {
 			Game::logger->Log("MasterServer", "Failed to convert fdb to sqlite");
 			return EXIT_FAILURE;
 		}
+	} else if (!std::filesystem::exists(BinaryPathFinder::GetBinaryDir() / "resServer" / "CDServer.sqlite")) {
```

Comment:
I think this condition is a bit weird to reason about. Yes, if both don't exist, then it would have gone to the first if case, so if the sqlite in resServer does not exist, and we get here there must be a sqlite in res to move over.

But considering, that this is essentially a migration, I think this should just be a positive check for existance, nested under a simple "resServer sqlite missing" before converting from fdb